### DISCUSSION TITLE
Help porting mcu-backend to different display.

### DISCUSSION THREAD

**Discussion by JaminMartin** [slint-ui/slint#6936] [Q&A]
Hi, I had a hopefully quick question. 

I was wondering if there were any guides on how to port the pi-pico printer demo to a different display - or how to get started with it. 

The display I have is a [SPI TFT 3.5" LCD Touch Module ILI9488](https://robokits.co.in/displays/lcd-display/spi-tft-3.5-lcd-touch-module-ili9488-driver-480x320) and I would like to try get the printer demo running on it. The waveshare boards are limiting in their application scope so I would like to get the demo up and running on a detached screen. 

There is a [display driver](https://github.com/chrismoos/ili9486-driver?tab=readme-ov-file) for this display, so I can replace the `st7789` display driver, but I am unsure about the touch aspect of this. 

I am relatively new to embedded development so any guidance is much appreciated. 

Cheers!

**Comment by ogoffart** [comment]
The documentation/guide we have is on this page: https://docs.rs/slint/latest/slint/docs/mcu/index.html

But for the touch, you'll have to search online what is the touch controler and hope there is a driver for it. If there isn't, it is usually not too hard to write a touch driver if you find a C implementation, or a datasheet.
